### ISSUE TITLE
Research and Implement: Adding additional ADR data to the API

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/openFEC#4793]
**What we're after:**
Several pieces of ADR data are currently missing from the API. These are the items that are still needed:

- Complainant
- Referring office
- Case status
- Non-monetary terms/non-civil penalties
- Commission votes: commissioner names and vote
- Citations (lower priority)

**Completion criteria:**
- [ ] The items listed in this ticket are added to the API

**Commission votes on EQS:** Example: ADR #1
<img width="691" alt="Screen Shot 2021-04-15 at 3 47 51 PM" src="https://user-images.githubusercontent.com/31663028/114929594-fce5b300-9e01-11eb-88d1-910a87ff4d27.png">

**Designs:**
![ADR Canonical Copy 2](https://user-images.githubusercontent.com/31663028/114942903-370b8080-9e13-11eb-9b29-4e3097513f8f.jpg)

**Comment by rfultz** [comment]
Nudging to 14.4, waiting on information from contractor

**Comment by patphongs** [comment]
Waiting on response from ADR, moving forward to 14.6.

**Comment by patphongs** [comment]
Moving back to 14.5 as we have received a response from ADR

**Comment by fec-jli** [comment]
We already talked to the contractor on 05/06/2021. They will work on how to get the data:
Complainant
Referring office
Case status
Non-monetary terms/non-civil penalties
Commission votes: commissioner names and vote
Citations (lower priority)

**Comment by fec-jli** [comment]
Waiting on response from ADR Tiger team, moving forward to 15.3

**Comment by patphongs** [comment]
Moved to 16.1 while we wait for the data

**Comment by fec-jli** [comment]
By Paul message 08/04/2021:
we are moving the ADR work to the beginning of PI 16. In the next week or so, please set up a requirements meeting with Tygart, required/interested website team members and David G. Once Tygart has the requirements their goal will be to deliver the required changes by October 12.

**Comment by pkfec** [comment]
Get Commission votes for case_id=36100003125764

-- Get ADR case:
SELECT DISTINCT
        case_id,
        case_no,
        name,
        case_type,
        published_flg
    FROM fecmur.cases_with_parsed_case_serial_numbers_vw
    WHERE case_type = 'ADR'
    AND case_no = '974'
    

  
  -- Get Commission Votes:
SELECT * FROM fecmur.COMMISSION c, fecmur.votes v, fecmur.entity e
WHERE c.CASE_ID = 36100003125764
AND c.commission_id = v.commission_id
AND e.entity_id =  v.entity_id
ORDER BY c.vote_date desc;
